tests/test_drag_coefficient.py

```diff
@@ -0,0 +1,48 @@
+""" Test the drag coefficient calculation for different objects. """
+
+import os
+
+import inductiva
+import pytest
+
+import windtunnel
+
+
+@pytest.fixture(scope="module", name="setup_machine_group")
+def setup_machine_group_fixture():
+    mg = inductiva.resources.MachineGroup(machine_type="c2d-highcpu-32",
+                                          num_machines=1,
+                                          spot=True)
+    mg.start()
+
+    yield mg.name
+
+    mg.terminate()
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "filename, expected_drag_coeff",
```

Comment:
leave comment saying that these are not the "true coefficients", but for the moment is what we are getting with resolution 3.

Reply:
```suggestion
# Test the drag coefficient calculation for different objects
# The drag coefficients here are not the ground truth values.
# They are only an approximation for lower resolution simulations.
# And are only used for sanity checking the implementation.
```